internal/converter/types.go

```diff
@@ -360,7 +360,7 @@ func (c *Converter) convertMessageType(curPkg *ProtoPackage, msg *descriptor.Des
 		jsonSchemaType.AdditionalProperties = []byte("true")
 	}
 
-	c.logger.WithField("message_str", proto.MarshalTextString(msg)).Trace("Converting message")
+	c.logger.WithField("message_str", prototext.Format(msg)).Trace("Converting message")
```

Comment:
`prototext.Marshal` should already be the `proto.Message.String` implementation. Plus, `proto.Message.String` should or will [use the `debug_redact` field](https://github.com/golang/protobuf/issues/1655):
```suggestion
	c.logger.WithField("message_str", msg).Trace("Converting message")
```

Reply:
Done